meson.build

```diff
@@ -143,7 +143,6 @@ incdir = include_directories(['.', 'ccan', 'src'])
 
 ################################################################################
 sources = []
-subdir('ccan')
```

Comment:
IIRC we need this for the debug build. For this configuration the additional ccan/str/debug.c is necessary.